docs/tab-based-navigation.md

```diff
@@ -204,3 +204,6 @@ export default createBottomTabNavigator(
 It's common to attempt to use a standalone tab bar component without integrating it into the navigation library you use in your app. In some cases, this works fine! You should be warned, however, that you may run into some frustrating unanticipated issues when doing this.
 
 For example, React Navigation's `TabNavigator` takes care of handling the Android back button for you, while standalone components typically do not. Additionally, it is more difficult for you (as the developer) to perform actions such as "jump to this tab and then go to this screen" if you need to call into two distinct APIs for it. Lastly, mobile user interfaces have numerous small design details that require that certain components are aware of the layout or presence of other components &mdash; for example, if you have a translucent tab bar, content should scroll underneath it and the scroll view should have an inset on the bottom equal to the height of the tab bar so you can see all of the content. Double tapping the tab bar should make the active navigation stack pop to the top of the stack, and doing it again should scroll the active scroll view in that stack scroll to the top. While not all of these behaviors are implemented out of the box yet with React Navigation, they will be and you will not get any of this if you use a standalone tab view component.
+
+## A tab navigator contains a stack and you want to hide the tab bar on specific screens
+[See the documentation here](https://reactnavigation.org/docs/en/navigation-options-resolution.html#a-tab-navigator-contains-a-stack-and-you-want-to-hide-the-tab-bar-on-specific-screens)
```

Comment:
sorry if I didn't make myself clear, my intention was not only to use markdown for the link, but also *not* use an absolute link, kinda like in this snippet: `You may also use [createMaterialBottomTabNavigator](material-bottom-tab-navigator.html)` - this is taken from the top of this file.